edgexfoundry/app-functions-sdk-go
# feat: Add option for Batch to merge results before sending.

This is needed when batching Metric data that has been transformed to
Line Protocol format.

closes #1101

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

     TBD

## Testing Instructions
Build and run App Template service to generate metrics
Build ASC with this branch
Configure ASC metrics-profile with:
- `ExecutionOrder = "ToLineProtocol, Batch, HTTPExport"`
- `MergeOnSend = "true"` in Batch function parameters
- HTTP to send to InfluxDB
Run ASC with metrics-profile with DEBUG logging
Verify  HTTP Export reports `status is 204 No Content` in log file

## New Dependency Instructions (If applicable)
